src/github.com/couchbase/sync_gateway/db/changes.go

```diff
@@ -403,21 +422,31 @@ func (db *Database) MultiChangesFeed(chans base.Set, options ChangesOptions) (<-
 			default:
 			}
 
+			// Reset added channels
+			addedChannels = nil
+
 			// Before checking again, update the User object in case its channel access has
 			// changed while waiting:
+			base.LogTo("Changes+", "MultiChangesFeed checking for reload..................")
```

Comment:
Not sure if the ellipsis are going to provide any signal or just add noise.  (if you could animate it, that would be a different story!)